docker/Dockerfile

```diff
@@ -45,4 +45,4 @@ ENTRYPOINT ["dbt"]
 # dbt-core
 ##
 FROM base as dbt-core
-RUN python -m pip install dbt-teradata==1.3.3.0
+RUN python -m pip install dbt-teradata==1.3.3.1
```

Comment:
Good for now, but eventually, this should be managed by `bump2version` like any other place that uses the current release number.

Reply:
Sure Adam, will do that for next release.